tasks-by-users/nehalskdev/01 Project/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<title> Kraj Men's wear </title>
+<div>
+    <head>
+        <meta charset="utf-8">
+        <meta name="keywords" content="clothes, design, men fashion,dress,art">
+        <meta name="discription" content="style, fashion,art,sale,Formals">
```

Comment:
rewrite this code looking very bad @nehalskdev 
you have not followed proper structure of html document most of your content wont reflect on web page.

Reply:
can you suggest how i can rewrite it please.

Reply:
use prettier extension @nehalskdev

Reply:
its done now